ui/style.css

```diff
@@ -1563,6 +1563,40 @@ span.tag-input[contenteditable]:empty::before {
     object-fit: cover;
 }
 
+.convictions-known {

```

Comment:
You can actually do `.convictions-known, incidents-known` here and this will cover both elements. This way you avoid duplicate CSS and can delete the styling from line 1639 - 1644. The same goes for the other styling properties.

Reply:
done <3